release-deployment.md

```diff
@@ -138,20 +138,43 @@ There's nothing special about that. Each developer follows the above [Develop a
 1. Finish off the tasks in the release checklist. Once everything is done, close
    the release PR.
 
-**TBD: Discuss**
-Mike N: Long-lived release branches, yes/no?
-
-1. Create the release on Github from `master`.
+1. Delete the release branch on Github.
 
 ### Change in plan, pull a feature from a release
 
 **TBD: Discuss**
 Mike N: That probably means recreating the release branch, unless we have short-lived release branches
 
-### Change request
+### Supporting old releases
+
+In a release based project old releases can often need some maintenance. Critical bug fixes and strategic feature requests need to be supported on old releases. Old releases often become incompatible with the most recent versions of projects.
+
+In order to support old release versions gitflow has introduced the concept of support branches. Support branches are long living branches created to support major or minor versions of the project. Support branches do not get merged back into `master` or `develop` (this would cause major merge issues which are time consuming and error prone if attempted). Instead commits can be cherry picked from the support branch back into `develop`. 
+
+Support branches can be thought of as the `master` branch for old releases. Support branches for major releases should be named as `support-v<major>.x`. Support branches for minor releases should be named as `support-v<major>.<minor>.x`.
+
+Here is an example of creating a support branch for v1.0 assuming v2.0 of the project has already been released.
+
+1. Create the support branch and release branch for the patch release.
+
+    ```
+    git checkout v1.0.0
```

Comment:
Maybe this is obvious to others, but it took me a second here to see that this is checking out a tag instead of a branch. Maybe we should add a comment explaining that we are checking out the release tag for the creation of the branch?